meetings/2024-03-06.md

```diff
@@ -45,12 +50,93 @@ URL: https://github.com/solid/specification/discussions/618
 
 ## Topics
 
+### Status of the WG Charter
+* HZ: PAC, any updates?
+* PAC: The W3C Council that decides on FO's has started. Not sure how long it will take. The Council re recent Process is relatively new. The Team's recommendation was to submit a new charter. Moving with the existing charer is inappapropiate and that TAG also raised some concerns. When we submit a new charter that tells us objections that can be addressed in the new charter. Some of them may be raised again. Not all objections are 'justified'. We can argue against those. If they come up again, we can go to the Council again. We know these objections would come up. Then the home that the next Council can overrule them again if they come up.
+* TBL: With one of the objectors I had back and forth. Their issues could be resolvable. The name of the group changing being larger.
```

Comment:
```suggestion
* PAC: The W3C Council that decides on FO's has started. Not sure how long it will take. The Council Process is relatively new. The Team's recommendation was to submit a new charter. Moving with the existing charter is inappropriate, and the TAG also raised some concerns. That tells us objections that can be addressed in the new charter. Some of them may be raised again. Not all objections are 'justified'. We can argue against those. If they come up again, we can go to the Council again. We knew some objections would come up. The hope is that the next Council will overrule them again if they come up again.
* TBL: I had back and forth with one of the objectors. Their issues could be resolvable. Changing the name of the group is larger.
```